filebeat/templates/deployment.yaml

```diff
@@ -0,0 +1,141 @@
+# Deploy singleton instance in the whole cluster for some unique data sources, like aws input
+{{- if .Values.deployment.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: '{{ template "filebeat.fullname" . }}'
```

Comment:
these single quotes make it harder to compare daemonset and deployment templates using some `diff daemonset.yaml deployment.yaml` or editor diff options. 

Is there any specific reason for adding them here?

Could you stick to `daemonset.yml` "format" so a diff between the 2 templates should print only the differences?